framework/doc/content/source/outputs/png/PNGOutput.md

```diff
@@ -0,0 +1,18 @@
+# PNGOutput (Outputs)
+
+The PNGOutput object is the class for writing png files from the input files.
+
+Following is an example of all the different input variables that can be used.
+
+```
+[Outputs]
+  [png]
+    type = PNGOutput
+    transparent = true    #indicates whether the background will be transparent
+    resolution = 50       #resolution of each point
```

Comment:
Just looking at the documentation strings I have no idea what this number means. The code suggests it is  a scale factor to get from MOOSE length units to pixels. I suggest to make this clear in the docs.

Reply:
The resolution might be "pixels", except that's not even a great definition with things like retina displays and everything else that complicates rendering sizes. This is the number of "samples" taken in the x and y directions right now but that's not the best solution for resolution, which should be in units per something. Also we need to account for different spacing in x and y. More work to do.